client/src/cmdhfmf.c

```diff
@@ -4187,14 +4196,15 @@ static int CmdHF14AMfCSetUID(const char *Cmd) {
 
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "hf mf csetuid",
-                  "Set UID, ATQA, and SAK for magic gen1a card",
+                  "Set UID, ATQA, and SAK for magic gen1a card\n"
+                  "Only works with magic gen1a cards\n",
```

Comment:
same cake on cake feeling.  First state that its for Gen1a,   then express that once more in the next line